Project/register.html

```diff
@@ -12,215 +12,17 @@
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 		<![endif]-->		
 	</head>
-
-	<body>
-		<div id="menu" class="menu"></div>
+	
+	<body onload="resetForm()">			
+		
+		<div id="menu" class="menu"></div><!-- menu.js: displays the menu here -->
 		<script src="menu.js"></script>
 			
 		<h1>Register</h1>	
-			
-		<div id="formdiv" class="register">
-			<form onsubmit="submitForm()">
-				<table>
-					<thead>
-						<th colspan="3">New User Registration</th>
```

Comment:
Might need you to explain whats going on with the <thead> <tr> <td> .

Reply:
thead/ th/ tr

Reply:
table layout for the form https://www.w3schools.com/html/html_tables.asp